component/admin/language/en-GB/en-GB.com_redshop.ini

```diff
@@ -3337,6 +3340,7 @@ COM_REDSHOP_ORDERID = "Order ID"
 COM_REDSHOP_ORDERNUMBER = "Order Number"
 COM_REDSHOP_FULLNAME = "Name"
 COM_REDSHOP_USERNAME = "User name"
+COM_REDSHOP_PASSWORD = "Pass word"
```

Comment:
@hoangvan149dhv typo mistake: correct is password

Reply:
@anhredweb  Hi anh, could you please check again. I fixed